Loop/Managers/DeviceDataManager.swift

```diff
@@ -193,7 +195,7 @@ class DeviceDataManager: CarbStoreDelegate, TransmitterDelegate, ReceiverDelegat
         }
 
         // Trigger device status upload, even if something is wrong with pumpStatus
-        remoteDataManager.uploadDeviceStatus(pumpStatus)
+        nightscoutDataManager.uploadDeviceStatus(pumpStatus)
```

Comment:
Would be nice to follow up later and move lines 184-195 into `uploadDeviceStatus`